### DISCUSSION TITLE
there is a snapshot file with the modification time of xxx available would you like to load it?

### DISCUSSION THREAD

**Discussion by saadxprt** [freerouting/freerouting#227] [Q&A]
i want to disable this feature 
![image](https://github.com/freerouting/freerouting/assets/96834166/282c0fb1-6c24-44bc-b1d9-71d8a15702f7)

im running free-routing with 
`java -jar freerouting-1.8.0.jar  -de ./MyBoard.dsn -do ./MyBoard.ses -dl `

how do i disable this feature? im running an automation, so unless i press on yes or no it will not work

**Comment by andrasfuchs** [comment]
You can't disable this feature at the moment, but you can delete the snapshot file(s) before running freerouting to prevent this dialog to show up.

I think you have a valid point, this dialog should have at least a timeout, defaulting to "no", or we could have a command line argument that makes freerouting ignore the snapshot files.

**Comment by andrasfuchs** [comment]
I created an issue for this here: https://github.com/freerouting/freerouting/issues/228, reply there, please.
